### ISSUE TITLE
Programmer should not be forced to click on method or class definition

### ISSUE THREAD

**Issue by Ducasse** [pharo-ide/Calypso#345]
to define a method or a class.
This is a source of many many mistakes.

We should have one pane where we can paste a class definition or remove a class definition
and add a method definition and compile it. 

NO MODE!!!!!

**Comment by JurajKubelka** [comment]
I prefer having separate pane for class definition, for defining new method, for changing selected method.

What might be more apparent (I do not know how) is defining traits. Currently it is in a context menu as a modal dialog.

**Comment by Ducasse** [comment]
This is not the point to have or not a pane. this is to have tons of errors because people should plan and click on the right pane. SO THIS SHOULD CHANGE! and it will change!

**Comment by Ducasse** [comment]
From that perspective I'm fed up that people do not listen to me when this is about working flow. 
I spent days with students learning Pharo.

**Comment by bencoman** [comment]
@Ducasse, So what you are saying is that the multiple tabs is okay, but that each tab should accept both class and method definitions? (except the comment tab I guess)

**Comment by Ducasse** [comment]
Yes and in addition we should be able to see the class comments else what is the point.
